testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/concurrency/ConcurrencyTest.java

```diff
@@ -51,6 +62,38 @@ public void concurrentTest(KeycloakRunnable... tasks) throws Throwable {
         System.out.println("took " + end + " ms");
     }
 
+    // KEYCLOAK-8141 Verify that no attribute values are duplicated, and there are no locking exceptions when adding attributes in parallell
+    @Test
+    @Ignore
+    public void createUserAttributes() throws Throwable {
+        AtomicInteger c = new AtomicInteger();
+
+        UsersResource users = testRealm().users();
+
+        UserRepresentation u = UserBuilder.create().username("attributes").build();
+        Response response = users.create(u);
```

Comment:
The response is never closed and causes connection leak.

`Creator` relieves you from boilerplate code, see e.g. https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/UserTest.java#L2047.

Reply:
Doesn't really save me anything - just closing the response. IMO the try with makes the code less readable and messy.